scss/widgets/generic/calendar/_index.scss

```diff
@@ -226,6 +226,19 @@ $generic-calendar-height: $generic-calendar-body-height + $generic-calendar-navi
       }
     }
   }
+
+  &.dx-calendar-range-date:not(.dx-calendar-selected-date) {
+    &,
+    &.dx-calendar-today {
+      color: $base-inverted-text-color;
+      box-shadow: inset 0 0 0 1000px lighten($calendar-cell-selected-bg, 30%);
```

Comment:
Why we use 1000px here?

Reply:
I have no idea actually, for now generic range styles are copypaste of generic selected styles, but with a lightened color. As far as generic design of DateRangeBox is not ready, I don't see any point of concentrating attention on temporary styles